files/amazon-2/sge_preinstall.sh

```diff
@@ -1,6 +1,20 @@
-#!/bin/sh
+#!/bin/sh -x
```

Comment:
what about using /bin/bash so that we pick a specific implementation and do not rely on what is symlinked by sh? Also do we want to keep the -x?

Reply:
Yeah it should definitely be bash. That was an oversight on my part.

Reply:
Done.